ceres.py

```diff
@@ -677,6 +679,8 @@ def read(self, fromTime, untilTime):
     byteRange = pointRange * DATAPOINT_SIZE
     packedValues = fileHandle.read(byteRange)
 
+    fileHandle.close()
```

Comment:
This whole section could be refactored to use `with open` to ensure the files are closed when we are finished with them.

```
     with open(self.fsPath, 'rb') as fileHandle:
       fileHandle.seek(byteOffset)
  
       timeRange = int(untilTime - fromTime)
       pointRange = timeRange // self.timeStep
       byteRange = pointRange * DATAPOINT_SIZE
       packedValues = fileHandle.read(byteRange)
```